src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -857,6 +857,12 @@ dmaTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptr
 
     std::stringstream run_details;
     size_t block_size = 16 * 1024 * 1024; // Default block size 16MB
+
+   // check if the bank has enough memory to allocate
```

Comment:
@AShivangi I am not sure we accept this type of comment, should we use /* for multiple lines?

Reply:
No, according to C++ standards, we should have "//" comments even for multiple lines